prophyc/generators/schema.py

```diff
@@ -0,0 +1,134 @@
+from collections import namedtuple
```

Comment:
Since there is `parsers/prophy.py`, I think there should be `generators/prophy.py` as well - if only for consistency. The word `prophy` is used as a name of language, not as a name of framework, just like protobuf has two meanings (language/framework). The word `schema` is controversial, because it has even more meanings: both C++ and ISAR XML serve as prophy schemas.